libs/core/src/components/FormElementWrapper.ts

```diff
@@ -1,10 +1,35 @@
-import {type Component, defineComponent, h, type PropType, provide, Teleport} from 'vue';
-import {type AnyElementInstance, type FormInstance} from '../types';
+import {type Component, defineComponent, h, type PropType, provide, Teleport, type VNode} from 'vue';
+import {isOfType} from '@myparcel/ts-utils';
+import {type AnyElementInstance, type ComponentOrHtmlElement, type FormInstance} from '../types';
 import {INJECT_ELEMENT} from '../symbols';
 import {useTestAttributes} from '../composables';
 import FormElement from './FormElement.vue';
 
 // noinspection JSUnusedGlobalSymbols
+type RawSlots = {
+  [name: string]: unknown;
+  $stable?: boolean;
+};
+
+const convertSlots = (component: ComponentOrHtmlElement): RawSlots => {
```

Comment:
```suggestion
/**
 * @TODO: check if this can be removed before releasing 1.0.0
 */
const convertSlots = (component: ComponentOrHtmlElement): RawSlots => {
```